### PR TITLE
Wr sprint2

### PR DESCRIPTION
**Pull request by RozwalkaWojciech**

### CODE REVIEW

File: src/main/java/com/infoshare/drinkywinky/menu/Menu.java [hunk 1]
```diff
@@ -40,36 +40,36 @@ public static void mainMenu(){
...
-            switch (Choicemenu.choiceMenu()) {
+            switch (ChoiceMenu.choiceMenu()) {
                 case 1:
                     stdout.info(" CHOSEN : 1. List of all recipes   \n");
-                    new drink.menu.ShowByCategory().showByCategory();
+                    new ShowByCategory().showByCategory();
                     //TODO PRINT ALL EVENTS
                     break;
                 case 2:
                     stdout.info(" CHOSEN : 2. Search for specific recipe\n");
```
Comment (radoslaw-piwowarczyk) [inline src/main/java/com/infoshare/drinkywinky/menu/Menu.java]: Line 50: (Missing "a" before "specific)

File: src/main/java/com/infoshare/drinkywinky/menu/ShowByCategory.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ void showByCategory() {
             stdout.info("\n");
             stdout.info("CHOOSE NUMBER: \n");
```
Comment (radoslaw-piwowarczyk) [inline src/main/java/com/infoshare/drinkywinky/menu/ShowByCategory.java]: Line 28: (Missing "a" before "NUMBER"

File: src/main/java/com/infoshare/drinkywinky/menu/Menu.java (hunk 1 above)
Comment (radoslaw-piwowarczyk) [inline src/main/java/com/infoshare/drinkywinky/menu/Menu.java]: Line 50: (Missing "a" before "specific)

File: src/main/java/com/infoshare/drinkywinky/menu/Menu.java [hunk 3]
```diff
@@ -40,36 +40,36 @@ public static void mainMenu(){
...
-                    new drink.menu.ShowByCategory().showByCategory();
+                    new ShowByCategory().showByCategory();
                     //TODO metoda 2
                     break;
                 case 3:
                     //TODO metoda 3
                     stdout.info(" CHOSEN : 3. Add/remove recipe  \n");
```
Comment (radoslaw-piwowarczyk) [inline src/main/java/com/infoshare/drinkywinky/menu/Menu.java]: Line 56: (Missing "a" before "recipe")

File: src/main/java/com/infoshare/drinkywinky/menu/Menu.java [hunk 4]
```diff
@@ -40,36 +40,36 @@ public static void mainMenu(){
...
                     break;
                 case 4:
                     //TODO metoda 3
                     stdout.info(" CHOSEN : 4. Manage yours favorites \n");
-                    new drink.menu.ShowByCategory().showByCategory();
+                    new ShowByCategory().showByCategory();
                     break;
                 case 5:
                     //TODO metoda 3
                     stdout.info(" CHOSEN : 5. Search for drink by category\n");
```
Comment (radoslaw-piwowarczyk) [inline src/main/java/com/infoshare/drinkywinky/menu/Menu.java]: Line 66: (missing "a" before "drink" -> Search for a drink by category.

**Review by radoslaw-piwowarczyk — APPROVED** [review]

**Review by DanielLaga84 — APPROVED** [review]
Approved.

**Pull request merged**